bin/main

```diff
@@ -1,70 +1,121 @@
 #!/usr/bin/env ruby
+
 # rubocop: disable Style/StringLiterals
 
-# Welcome the player to the game
+# rubocop: disable Style/GuardClause
+
+# rubocop:disable Lint/UnreachableCode
+
+# rubocop: disable Style/ConditionalAssignment
+
+# rubocop: disable Style/MultipleComparison
+
+require_relative '../lib/board.rb'
+require_relative '../lib/player.rb'
+
+def get_choice(arg)
+  loop do
+    puts 'Please enter a number between 1 to 9: '
+    choice = gets.chomp.to_i
+    index = choice - 1
+    if arg.include? choice
+      return index
+      break
+    else
+      puts 'Wrong position, please choose another one'
+    end
+  end
```

Comment:
That snippet must be moved into the Board class as method returning true/false indicating whether or not the number entered by the user is a good position or not
```rb
def good_position?(choice)
    index = choice - 1
    if this.array.include? choice # array = [1, 2, 3, ..., 9]
      return index
    else
      return false
    end
end
```

Reply:
:100: